internal/integration/publish_test.go

```diff
@@ -45,74 +48,76 @@ func TestPublish(t *testing.T) {
 	db := env.Database()
 	enClient := &EnServerClient{client: client}
 
+	exportPeriod := 2 * time.Second
+	criteria := publishdb.IterateExposuresCriteria{
+		OnlyLocalProvenance: false,
+	}
+
 	// Create an authorized app.
-	aa := env.AuthorizedAppProvider()
-	if err := aa.Add(ctx, &authorizedappmodel.AuthorizedApp{
-		AppPackageName: "com.example.app",
-		AllowedRegions: map[string]struct{}{
-			"TEST": {},
-		},
-		AllowedHealthAuthorityIDs: map[int64]struct{}{
-			12345: {},
-		},
+	startAuthorizedApp(ctx, env, t)
 
-		// TODO: hook up verification, and disable
-		BypassHealthAuthorityVerification: true,
+	// Create a signature info.
+	createSignatureInfo(ctx, db, exportPeriod, t)
+
+	firstPayload := newPayloads(3)
+	enClient.PublishKeys(t, firstPayload)
+	assertExposures(t, ctx, db, 3)
+	firstBatchKeys := sets.NewString()
```

Comment:
I really don't want to pull in apimachinery for this. Can we just use a map or string slice?

Reply:
Sure, I can do that

Reply:
Done